docker/build.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
I would prefer this file (build.sh) to be in the root, not in a "docker" folder. The script isn't specific for anything Docker. You can rust it locally as well.

Reply:
It's meant to run in the Docker container though, unless you happen to have the exact same configuration. For instance, my computer doesn't have any `/opt/rh` folder. And my computer at home isn't even able to run .sh scripts because it's Windows.

Reply:
After discussing this in person, I hope you can reconsider this, based on the following points:

* This will only run on CentOS because of the tools in `/opt/rh` and the `devtools-7` package. Nobody I know of in our office (or in general really) develops on CentOS. Putting a `build.sh` script in root could be misleading if it only works on CentOS.
* The script is developed for Docker and we plan to maintain it only for Docker, so that it can be kept more stable.

Reply:
Yes, we discussed and I agree on the current file path.